From: conda-forge <conda-forge@users.noreply.github.com>
Subject: [PATCH] Fix ISAAC prototypes for GCC 15

Backport the two-line fix from upstream PR #27:
https://github.com/umontreal-simul/TestU01-2009/pull/27

GCC 15 rejects the old K&R-style declarations because the calls provide
arguments that are not present in the declarations.

---
 testu01/ucryptoIS.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/testu01/ucryptoIS.c b/testu01/ucryptoIS.c
--- a/testu01/ucryptoIS.c
+++ b/testu01/ucryptoIS.c
@@ -131,9 +131,9 @@ typedef  struct randctx  randctx;
  If (flag==TRUE), then use the contents of randrsl[0..RANDSIZ-1] as the seed.
 ------------------------------------------------------------------------------
 */
-static void randinit(/*_ randctx *r, word flag _*/);
+static void randinit(randctx *r, word flag);
 
-static void isaac(/*_ randctx *r _*/);
+static void isaac(randctx *r);
 
 /*
 ------------------------------------------------------------------------------
-- 
2.47.0
